content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -42,10 +42,10 @@ refs/tags/baseline-experiment:
         cnn-64
 ```
 
-This command lists the experiments originated from `HEAD`. You can add any other
-options to the remote command, including `--all`. (see previous section).
+This command lists remote experiments originated from `HEAD`. You can add any
+other options to the remote command, including `--all` (see previous section).
```

Comment:
> From https://github.com/iterative/dvc.org/pull/2862#pullrequestreview-798142029

How is `HEAD` is determined for a remote to list experiments from (e.g. `dvc exp list origin`)? Is it the local repo's HEAD? Is it the default branch HEAD always? We should probably clarify. Cc @iesahin do you know?

Reply:
Ping @iesahin

Reply:
Or, I guess @pspeter can also confirm. Thanks

Reply:
I thought I've replied this, sorry. 

It's the `HEAD` of the remote repository. So when `dvc exp list origin` it lists the `HEAD` of the `origin` even if we are on another branch.

Reply:
I see... OK, updated.